tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -582,41 +580,6 @@ describe(__filename, () => {
     );
   });
 
-  it('omits version history for search plugins', () => {
-    // Search plugins only have one listed version so showing their
-    // version history is useless–we just omit it.
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      type: ADDON_TYPE_OPENSEARCH,
-    });
-    const root = render({ addon });
-
-    expect(root.find('.AddonMoreInfo-version-history')).toHaveLength(0);
-  });
-
-  it('omits version history for search tool', () => {
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      type: ADDON_TYPE_OPENSEARCH,
-    });
-    const root = render({ addon });
-
-    expect(root.find('.AddonMoreInfo-version-history')).toHaveLength(0);
-  });
-
-  it('links to version history if add-on is a theme', () => {
```

Comment:
I think we should keep this test case.

Reply:
Alright. I added it back, and it did not break any tests.